### PR TITLE
Fork helm-operator adjustments for CI and documentation

### PR DESCRIPTION
**Pull request by SimonBaeumer**
CI migration of the helm-operator fork. 
Did not deleted the original deploy actions.

 - Disable some docker image push
 - Execute CI on all pushes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,41 @@
 # helm-operator
 
-[![Build Status](https://github.com/joelanford/helm-operator/workflows/CI/badge.svg?branch=master)
-[![Coverage Status](https://coveralls.io/repos/github/joelanford/helm-operator/badge.svg?branch=master)](https://coveralls.io/github/joelanford/helm-operator?branch=master)
+[![Build Status](https://github.com/stackrox/helm-operator/workflows/CI/badge.svg?branch=main)
 
 Experimental refactoring of the operator-framework's helm operator
+
+### Why a fork?
```
Comment (SimonBaeumer) [inline README.md]: @gaurav-nelson Do you have any feedback for me here?
I would like to say that this is a fork which diverges in its use-case from the original operator implementation.

**Review by msugakov — APPROVED** [review]
I guess it would be good to have some convention how to label commits for easier upstreaming our changes. For example, this commit must not be picked for upstreaming and can be marked somehow like `[not for upstream] Fork helm-operator adjustments for CI and documentation`.
What do you think?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,11 +1,7 @@
 name: CI
 
 on:
-  push:
-    branches:
-      - '**'
-  pull_request:
-    branches: [ main ]
+  - push
```
Comment (misberner) [inline .github/workflows/ci.yml:4]: Curious as to why you changed this?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,41 @@
...
+
+As the helm-operator is an experimental refactoring and not actively maintained we started a fork to 
+further support [hybrid operators](https://github.com/operator-framework/operator-sdk/issues/670) based on Helm.
+
+This fork should used as a library and not is recommended to watch CustomResources by configured `watches`.
```
Comment (misberner) [inline README.md]: I don't understand this sentence, or how "watching CustomResources by configured watches" relates to whether or not it's being used as a library

File: README.md [hunk 4]
```diff
@@ -1,6 +1,41 @@
...
+
+### Quickstart
+
+Add this lib as a replace directive to your `go.mod`:
+
+```
+replace(
```
Comment (misberner) [inline README.md]: super nit: it's common to just provide a CLI command, e.g., `go mod edit -replace=github.com/joelanford/helm-operator=github.com/stackrox/helm-operator@main` (also, the line as it's written is invalid, as you need to specify a revision for the replacement module)

File: .github/workflows/ci.yml (hunk 2 above)
Comment (SimonBaeumer) [reply .github/workflows/ci.yml:4]: Because the CI was not triggered in the default setting.

File: README.md (hunk 4 above)
Comment (SimonBaeumer) [reply README.md]: done

**Comment by SimonBaeumer** [comment]
> I guess it would be good to have some convention how to label commits for easier upstreaming our changes. For example, this commit must not be picked for upstreaming and can be marked somehow like `[not for upstream] Fork helm-operator adjustments for CI and documentation`.
> What do you think?

👍 
Added labels for it, by default `upstream-triage` is added to indicate that it should be decided if this should be an upstream fix too.

File: README.md (hunk 3 above)
Comment (SimonBaeumer) [reply README.md]: Rephrased the whole paragraph.

**Review by misberner — APPROVED** [review]

**Pull request merged**